pkg/envtest/envtest_suite_test.go

```diff
@@ -35,12 +37,61 @@ var env *Environment
 var _ = BeforeSuite(func(done Done) {
 	logf.SetLogger(zap.LoggerTo(GinkgoWriter, true))
 	env = &Environment{}
+	initializeWebhookInEnvironment()
```

Comment:
we should either have a note here ("needs to be done here because it has to be done during env setup") or call the relevant install function separately during the test that needs it.

Reply:
my reasoning for having this here is that it tests the envtest initialization as a sideproduct (and the install function). If I did it in the test itself, I would not test that part. I'll put a not here